src/lib.rs

```diff
@@ -441,6 +441,21 @@ impl Rng {
         }
     }
 
+    /// Fill a byte slice with random data.
+    #[inline]
+    pub fn fill(&self, slice: &mut [u8]) {
+        // Filling the buffer in chunks of 8 is much faster.
+        let mut chunks = slice.chunks_exact_mut(8);
+        for items in chunks.by_ref() {
+            let r = self.u64(..);
+            items.copy_from_slice(&r.to_le_bytes());
```

Comment:
I think we should use `to_ne_bytes()` instead in order to avoid a performance hit on big endian systems.